django_irods

```diff
@@ -1 +1 @@
-Subproject commit 64cc56f3feb2617fc2fff1f3256e56ff96da319d
+Subproject commit b02faaaf759d9c78885b24c8bdd6bd040be86844
```

Comment:
@xuezhaokun You can cd to django_irods directory, do a ```git checkout master``` to pull in latest code which should point to the latest commit which develop branch points to, then you can cd to hydroshare directory and do a ```git add django_irods``` then push the change. That should get your branch in sync with develop branch for django_irods submodule.